src/Hl7.FhirPath.Tests/Functions/OperationsTests.cs

```diff
@@ -22,6 +22,19 @@ public class OperationsTests
         private static IEnumerable<object[]> EqualityOperatorTestcases() =>
             new (string expression, bool expected, bool invalid)[]
                  {
+                    ("(@2012 + 1 year) = @2013", true, false),
+                    ("(@2012-02 + 1 year) = @2013-02", true, false),
+                    ("(@2012-02-13 + 13 month) = @2013-03-13", true, false),
+                    ("(@2012 + 1 month) = @2012", true, false),
+                    ("(@2012 + 12 month) = @2013", true, false),
+                    ("(@2012-02-27 + 5 days) = @2012-03-03", true, false),
+
```

Comment:
Interesting, the 365 test doesn't pass there. Which means the .net DateTimeOffset class does it differently?

Reply:
https://www.timeanddate.com/date/dateadded.html?d1=01&m1=01&y1=2016&type=add&ay=&am=&aw=&ad=365&rec=
This isn't making sense to me why it should move up to the next year.
Where is the standard year defined? and is there a standard month defined too?

Reply:
The documentation for this testcase says:

"The first example above returns @2015, because 23 months only constitutes one year. The second example returns 2017 because even though 2016 is a leap-year, the time-valued quantity (365 days) is converted to 1 year, a standard calendar year of 365 days."

Reply:
Have asked on zulip

Reply:
This is because of the way Grahame (but probably Bryn in the CQL spec) has described datetime operations:

> For partial date/time values where the time-valued quantity is more precise than the partial date/time, the operation is performed by converting the time-valued quantity to the highest precision in the partial (removing any decimal value off) and then adding to the date/time value.

Reply:
This for sure does not align with what the operations on DateTimeOffset do.

Reply:
Happy with the explanation on zulip, and see the full table. Will update the implementation to cover that. (also reported it for the fhirpathjs implementation)

Reply:
Now done.